### ISSUE TITLE
Better user cache.

### ISSUE THREAD

**Issue by SilverCory** [minecraftly/minecraftly#27]
So one of the biggest things I've been running into is that redis isn't the best caching for user data, for example, UUID <-> name and (UUID|name)->Skin data

And lots of other stuff. Mostly this would be accessed by Bukkit, so MySQL? To add http into the mix is just going to add in another jump and still going to cause more issues than it will resolve.

My thoughts are mongo, maybe..

**Comment by SilverCory** [comment]
A few issues that relate to this matter.

#21 List of banned, trusted and muted players.
#20 Remove "Loading world...." we only really get the UUID on bukkit's side, so in order to say "Loading {player name}'s world" we need to fetch the name.

**Comment by viet** [comment]
I agree. MongoDB in addition to Redis would be a great combination each serving different purposes.

**Comment by SilverCory** [comment]
@viet is that a yes implement it, because it would make things a lot easier with several of these features and open the doors to several more.

**Comment by viet** [comment]
Of course! @SilverCory implement it.

How would it affect the time to production of our software? Any delay in days/weeks/months?

**Comment by SilverCory** [comment]
Ppft, a day at most once I start.
